### ISSUE TITLE
Conflict with commands in laptop layout

### ISSUE THREAD

**Issue by nvdaes** [CyrilleB79/NVDA-Dev-Test-Toolbox#1]
The following keystrokes are used in laptop keyboard layout by NVDA, and shouldn't be overriden by add-ons unless it's intentional, what seems not to be in this case:

- NVDA+shift+left arrow: in NVDA, navigate to previous object.
- NVDA+shift+rightarrow: in NVDA, navigate to next object.
- NVDA+right arrow: in NVDA, moves the review cursor to next character.

Ideally, I imo, this should be fixed before the add-on is included on the website since this affects the user experience of NVDA.
Cheers

**Comment by CyrilleB79** [comment]
Hi Noelia
Thanks Noelia.
Yes, I agree with you. Just after posting on the list, I had already in mind to unassign these gestures and let the user assign his own ones. In any case (even in the desktop layout), these gestures seem too common for a so specific feature.
I will take care to unassign them before submitting the add-on for the community website.

For the explanation, some features of this add-on were initially just scratchpatd scripts for my own use. When I have converted them to an add-on, I have  forgotten some points that were adapted for my personal use but not for other users.

**Comment by CyrilleB79** [comment]
Yes, I will do it myself with other slight modifications (e.g. use the add-on's category for these scripts)

**Comment by CyrilleB79** [comment]
Thanks @nvdaes.
This has been fixed in 109bba22f6f83cb9daf386b2b1903725d5273718: I have unassigned the gestures by default to let the user choose the one he wants to use.

**Comment by nvdaes** [comment]
Thanks, looking forward to see this on the website. 😃

2022-07-12 10:00 GMT+02:00, Cyrille Bougot ***@***.***>:
